book/01-introduction/sections/what-is-git.asc

```diff
@@ -84,8 +84,8 @@ Git has three main states that your files can reside in: _modified_, _staged_, a
 
 This leads us to the three main sections of a Git project: the working tree, the staging area, and the Git directory.
 
-.Working tree, staging area, and Git directory.
-image::images/areas.png["Working tree, staging area, and Git directory."]
+.Working tree, staging area, and Git directory
+image::images/areas.png[Working tree, staging area, and Git directory]
```

Comment:
The image will not be rendered in the PDF version. Please, return back double quotes here.

Reply:
> The image will not be rendered in the PDF version. Please, return back double quotes here.

Okay, I'll discard the whole https://github.com/progit/progit2/pull/1444/commits/d39824ab56d10fa031a985a7dd1fb953947e1ad9 commit.